tests/vmi_multus_test.go

```diff
@@ -512,6 +386,181 @@ var _ = Describe("Multus Networking", func() {
 	})
 })
 
+var _ = Describe("SRIOV", func() {
+
+	flag.Parse()
+
+	virtClient, err := kubecli.GetKubevirtClient()
+	tests.PanicOnError(err)
+
+	var vmiOne *v1.VirtualMachineInstance
+
+	tests.BeforeAll(func() {
+		tests.SkipIfNoMultusProvider(virtClient)
+		tests.BeforeTestCleanup()
+		result := virtClient.RestClient().
```

Comment:
I think you can remove this from here

Reply:
I've removed creation of non SR-IOV networks. I don't think I can remove the whole section since it prepares SR-IOV networks for test cases.